nadya/first/task3.py

```diff
@@ -0,0 +1,9 @@
+"""
+дано рядок тексту
+вивести кожне друге слово з кінця
+"""
+text="Just always and always look around and act wisely"
+newlist=list(text.split(" "))
+rang=len(newlist)
+for i in range(rang-2, -1, -2):
+    print(newlist[i])
```

Comment:
ok is everything
`def task2():
    text = "ghkdlh5673bjhdvfl"
    count = 0
    for i in text:
        if i.isdigit():
            count = count + 1
    print(count)`